db/migrate/20090821044418_add_scope_to_dependencies.rb

```diff
@@ -1,8 +1,8 @@
 class AddScopeToDependencies < ActiveRecord::Migration[4.2]
   def self.up
     add_column :dependencies, :scope, :string
-    Dependency.update_all(scope: "runtime")
-    announce "Please reprocess all gems after this migration"
+    # Dependency.update_all(scope: "runtime")
+    # announce "Please reprocess all gems after this migration"
```

Comment:
I generally think of old migrations as informational and not runnable. Are we really trying to keep them runnable?

Reply:
We currently do run them when testing the docker image, using db:create db:migrate, instead of db:prepare

Reply:
😮 Ok